### ISSUE TITLE
1px image import

### ISSUE THREAD

**Issue by Hohenheimsenberg** [UnderwaterApps/overlap2d#15]
I want to save some data in the scene, regarding to the scene and no to any object. So I tried to import an 1x1 alpha png with some variables, but it crashes.

This is the log:
```
java.lang.ArrayIndexOutOfBoundsException: 8
	at com.d.a.q.a(ResampleOp.java:422)
	at com.d.a.q.a(ResampleOp.java:152)
	at com.d.a.a.filter(AdvancedResizeOp.java:81)
	at com.uwsoft.editor.a.j.a(ResolutionManager.java:70)
	at com.uwsoft.editor.data.a.a.a(DataManager.java:1121)
	at com.uwsoft.editor.data.a.a.a(DataManager.java:1107)
	at com.uwsoft.editor.data.a.a.a(DataManager.java:59)
	at com.uwsoft.editor.data.a.j.run(DataManager.java:840)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by Hohenheimsenberg** [comment]
I have 4 different resolutions, so a 2x2 and 4x4 also makes it crash. But not a 8x8.

**Comment by azakhary** [comment]
Fixed in #16 

Instead of crash it will now give out a warning. The thing is Overlap2D is trying to resize your image to a smaller size for smaller resolution. So if you have 1x1 image then for smaller resolution it would be 0.5 pixel which is not an option. Even more then that the hard limitation on the library we are using to resize is 3 pixels, so it cannot go smaller then that. That means that normally, your smallest image should be bigger or equal then 3 * K where K is the ratio between your origin resolution and your smaller resolution.  I think for any practical implications this should be fine. And for some reason you need 1px image for all resolutions, it will not crash anymore, but give out a warning so you will be aware of the issue (the issue being you have image having same size for different resolutions)
